docs/_snippets/portable-stories-jest-override-globals.md

```diff
@@ -9,7 +9,7 @@ test('renders in English', async () => {
   const Primary = composeStory(
     PrimaryStory,
     meta,
-    { globals: { locale: 'en' } } // 👈 Project annotations to override the locale
+    { globals: { locale: 'en' } }, // 👈 Project annotations to override the locale
```

Comment:
This comma is also incorrect. The last entry in an object should have a trailing comma. The last argument passed into a function (like here) should not.